liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -210,5 +225,7 @@ void SourceReferenceFormatter::printErrorInformation(ErrorList const& _errors)
 
 void SourceReferenceFormatter::printErrorInformation(Error const& _error)
 {
-	printExceptionInformation(SourceReferenceExtractor::extract(m_charStreamProvider, _error));
+	printExceptionInformation(
+		SourceReferenceExtractor::extract(m_charStreamProvider, _error, Error::ErrorMessageDisclaimedWith::Severity)
```

Comment:
You don't really need `ErrorMessageDisclaimedWith` here. You could just use the overload that accepts severity directly:

```suggestion
		SourceReferenceExtractor::extract(m_charStreamProvider, _error, Error::errorSeverity(_error.type()))
```